tests/unit/test_download_query_api.py

```diff
@@ -30,7 +31,7 @@ def setUp(self):
         )
         # upload a file
         self.s3_filepath = "test-data/science_block_20221116_163611Z_idle.bin"
-        self.local_filepath = f"tests/unit/{self.s3_filepath}"
+        self.local_filepath = f"{self.s3_filepath}"
```

Comment:
Was this wrong before? Otherwise it seems like we shouldn't even need the variable `local_filepath` at all and this could be sent to the upload_file as `self.s3_filepath, self.bucket_name, self.s3_filepath`